doc/onlinehelp.rst

```diff
@@ -1,29 +1,92 @@
-:orphan:
-
 ###########
 Online Help
 ###########
 
 The **Online Help** pane provides a built-in web browser to explore dynamically generated Python documentation on installed modules—including your own—rendered by a ``pydoc`` server running in the background.
 
-.. image:: images/online_help/online_help_standard.png
-   :width: 570px
-   :alt: Spyder Online Help panel on the index page, a list of builtin modules
+.. image:: images/online_help/online-help-standard.png
+   :alt: Spyder Online Help pane on the index page, a list of builtin modules
 
 |
 
 
+
 =====================
 Using the Online Help
 =====================
 
-Enter the name of the item you'd like documentation on in the :guilabel:`Module or package` field, or the :guilabel:`Get` field, to load its doc directly.
-Or, if you're not sure of the name of the object you want help on, or are looking for a specific keyword, use the :guilabel:`Search` field to get a list of results.
-You can view and re-run previous searches with the drop-down menu or the :kbd:`Down Arrow` key in the :guilabel:`Module or package field`, navigate previous pages with the forward and back buttons (top left, left and right arrows), and go back to the homepage with the home button (house icon top left) or the ``Module Index`` link top right.
+The Online Help's home shows an index where you can browse the documentation of standard library modules, third party packages installed in your Python environment, and your own local code.
+Click on the name of any module to open its documentation. 
+
+.. image:: images/online_help/online-help-browse.gif
+   :alt: Spyder Online Help pane showing module browsing
+
+|
+
+Enter the name of the item you'd like help on in the :guilabel:`Package` field to load its documentation directly.
+
+.. image:: images/online_help/online-help-name.gif
+   :alt: Spyder Online Help pane showing module browsing by name
+
+|
+
+The module's file location is linked to the right of the doc's title, which you can click to view its source code.
+Standard library modules also have a link to the corresponding Python.org docs, which can be viewed right inside of the pane.
```

Comment:
Add a link to `Python.org docs` so users can click on it. Else it's unclear what docs you talking about.

Reply:
Will do.
https://docs.python.org/